app/frontend/app/templates/group.hbs

```diff
@@ -0,0 +1,146 @@
+              <h2 class="name">
+                {{model.name}}
+              </h2>
+              {{#if showEditMenu}}
+                {{{textarea class="edit-bio" rows="2" maxlength="140" placeholder="Briefly tell us about yourself..." value=model.truncatedBio}}}
+              {{else}}
+                <p class="group-bio">
+                  {{model.truncatedBio}}
+                </p>
+              {{/if}}
+            </div>
+          </div>
+
+          <div class="follow-widgets">
+            {{! Join/Leave Group Button }}
+            {{#if loading}}
+              <button class="btn btn-default btn-lg follow-button" disabled="disabled">
+                Loading... <i class="fa fa-spin fa-spinner"></i>
+              </button>
+            {{else}}
+              {{! edit button }}
+              {{#if isAdmin}}
+                {{#if showEditMenu}}
+                  <button class="btn btn-default btn-lg follow-button save edit-group" {{action "saveEditMenu"}}>
+                    <i class="fa fa-pencil"></i> Save
+                  </button>
+                {{else}}
+                  <button class="btn btn-default btn-lg follow-button edit edit-group" {{action "toggleEditMenu"}}>
+                    Edit Group
+                  </button>
+                {{/if}}
+              {{/if}}
+              {{#if currentMember}}
+                {{#if currentMember.pending}}
+                  <button class="btn btn-default btn-lg follow-button" disabled="disabled">
+                    Approval Pending...
+                  </button>
+                {{else}}
+                  <button class="btn btn-default btn-lg follow-button edit" {{action "leaveGroup"}}>
+                    Leave Group
+                  </button>
+                {{/if}}
+              {{else}}
+                {{#unless model.closed}}
+                  <button class="btn btn-default btn-lg follow-button" {{action "joinGroup" model}}>
+                    Join Group
+                  </button>
+                {{/unless}}
+              {{/if}}
+            {{/if}}
+
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+{{! Nav Bubbles }}
+<div class="profile-navigation">
+  <div class="container">
+    <div class="row">
+      <ul class="inline-list clearfix">
+        <li class="nav-link">
+          {{#link-to 'group.index'}}Feed{{/link-to}}
+        </li>
+        <li class="nav-link">
+          {{#link-to 'group.members'}}Members{{/link-to}}
+        </li>
+        {{!-- {{#if isAdmin}}
+          <li class="nav-link">
+            {{#link-to 'group.manage'}}Manage{{/link-to}}
+          </li>
+        {{/if}} --}}
+      </ul>
+    </div>
+  </div>
+</div>
+
+{{! Ad Banner }}
+<div class="container">
+  <div class="feed-head-advert">
+    {{ad-unit adId="1293413" adClass="257f81e798bd68dd81e60f42838f361f"}}
+  </div>
+</div>
+
+{{! Content }}
+<div class="min-height-wrapper">
+  {{#if model.closed}}
```

Comment:
If group closure has been removed then there is a ton of `{{group.closed}}` conditionals around in both the Handlebar templates, and Javascript that will need to be removed.